src/emulator/firestoreEmulator.ts

```diff
@@ -11,7 +11,8 @@ export interface FirestoreEmulatorArgs {
 }
 
 export class FirestoreEmulator implements EmulatorInstance {
-  static FIRESTORE_EMULATOR_ENV = "FIREBASE_FIRESTORE_EMULATOR_ADDRESS";
+  static FIRESTORE_EMULATOR_ENV = "FIRESTORE_EMULATOR_HOST";
+  static FIRESTORE_EMULATOR_ENV_ALT = "FIREBASE_FIRESTORE_EMULATOR_ADDRESS";
```

Comment:
I'm fine if you want to drop this, we're cutting a 7.0.0 release which seems like a good time to stop supporting it, your call

Reply:
That's ok it's no-cost for us to support it, I'd rather leave it.